Source/FaceFXEditor/Private/AssetTypeActions/AssetTypeActions_FaceFXBase.cpp

```diff
@@ -23,14 +23,14 @@
 #include "FaceFXEditorTools.h"
 #include "FaceFX.h"
 #include "Factories/FaceFXActorFactory.h"
-#include "Include/Slate/FaceFXResultWidget.h"
-#include "IMainFrameModule.h"
-#include "ModuleManager.h"
+#include "Include/�Slate/FaceFXResultWidget.h"
```

Comment:
This is causing a compile issue for me.

Is this the full / compiling code for MSVC?